### DISCUSSION TITLE
Add Flags of Countries

### DISCUSSION THREAD

**Discussion by maytham553-elite** [parfaitementweb/filament-country-field#14] [Ideas]
Hi,

I would like to request the addition of flags for each country in this plugin. Having the country flags would significantly enhance the usability and visual appeal of applications using this plugin.

**Comment by AlexisSerneels** [comment]
Hi! Thank you for the suggestion.
This is not something I would personally need so I won't dedicate time to it.
Feel free to submit a PR as I agree this would be a great enhancement and would merge it.

Cheers.

**Reply by maytham553-elite** [reply]
Thanks for your replay , Maybe I'll do it in my spare time.
